src/main/java/concatenatedwordschallenge/util/TaskFileReader.java

```diff
@@ -0,0 +1,46 @@
+package concatenatedwordschallenge.util;
+
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+public class TaskFileReader {
+
+    private static Logger logger = Logger.getLogger(TaskFileReader.class);
+
+    private List<String> words;
+
+    public TaskFileReader (String fileName) {
+        this.words = getWordsFromFile(getPathToResourcesFile(fileName));
+    }
+
+    private List<String> getWordsFromFile(String path) {
+        String wordsFromFile = "";
```

Comment:
```suggestion
        String wordsFromFile;
```

Reply:
This method requires this String to be previously initialized.
![image](https://user-images.githubusercontent.com/54272117/78063477-f02e2280-7398-11ea-97cd-b43f282858f3.png)